{"name": "测试数据：【 13200138000，123456，False，1007，用户不存在】", "status": "passed", "description": "该用例是针对获取用户登录接口的测试", "steps": [{"name": "步骤1 ==>> 登录用户", "status": "passed", "parameters": [{"name": "phone", "value": "'13200138000'"}], "start": 1628150803512, "stop": 1628150803512}], "attachments": [{"name": "log", "source": "c86d09ad-2598-4cad-9083-9f62ce154fb6-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "07828bfa-ddb2-47f5-ad46-bbe62eec0534-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "87c612fa-d072-4d28-8ddb-844c126c62e6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "phone", "value": "'13200138000'"}, {"name": "verificationCode", "value": "'123456'"}, {"name": "except_result", "value": "False"}, {"name": "except_code", "value": "1007"}, {"name": "except_msg", "value": "'用户不存在'"}], "start": 1628150803472, "stop": 1628150803512, "uuid": "c37f4046-f90d-40dd-a75b-1e43332c9b4f", "historyId": "3932cc63447aeccf4da8002ede2829a0", "testCaseId": "4cf2c2f7c5a680a41fe59cb327f3c0a1", "fullName": "testcases.api_test.test_kuai_login.TestUserLogin#test_login_user", "labels": [{"name": "story", "value": "用例--登录用户"}, {"name": "feature", "value": "用户登录模块"}, {"name": "severity", "value": "normal"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_kuai_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "zengxiaohua.local"}, {"name": "thread", "value": "10629-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_kuai_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}